### DISCUSSION TITLE
linux blockly issue

### DISCUSSION THREAD

**Discussion by WolfOptics** [BIPES/BIPES#78] [Q&A] [answered]
Hi sirs,
 I got some problem using blockly in linux system as shown as picture. 
[https://drive.google.com/file/d/1n4EhUH57qYAP2y8ppRB7OT9e7SsM2ooh/view?usp=sharing](url)

Could you help me? It was NOT workable in my raspberry pi 3B+.

Best Regards,

**Comment by rafaelaroca** [comment, marked as answer]
Hello. I tried to open the link you sent, but it is not available anymore. Could you give more details?

Did you download and run SocketServerWebREPL:

git clone https://github.com/rafaelaroca/SocketServerWebREPL.git
cd SocketServerWebREPL
python socketserverWebREPL.py

Like explained here:

http://bipes.net.br/docs/get-started/tech-details.html

Hope it helps.

best regards
Rafael
